Tests/RockLib.HealthChecks.AspNetCore.ResponseWriter.Tests/ResponseWriterTests.cs

```diff
@@ -145,29 +145,31 @@ public static class ResponseWriterTests
             .BeFalse();
     }
 
-    private static IEnumerable<JToken> GetHealthCheckEntries(string jsonBody) => 
-        JObject.Parse(jsonBody)!
+    private static IEnumerable<JToken> GetHealthCheckEntries(string jsonBody)
+    {
+        return JObject.Parse(jsonBody)!
             .SelectToken("checks")!
             .Children()
             .SelectMany(x => x.First!);
+    }
 
     private static async Task<string> GetJsonBody(DefaultHttpContext httpContext)
     {
         httpContext.Response.Body.Seek(0, SeekOrigin.Begin);
 
-        string bodyStr;
-        using (var reader = new StreamReader(httpContext.Response.Body))
-        {
-            bodyStr = await reader.ReadToEndAsync().ConfigureAwait(false);
-        }
+        using var reader = new StreamReader(httpContext.Response.Body);
+        var bodyStr = await reader.ReadToEndAsync().ConfigureAwait(false);
```

Comment:
Typically, in this case I wouldn't declare a variable and then return it. Just return the result of `ReadToEndAsync()` - same with what's done in `CreateHttpContext()`.

Reply:
yeah, I completely missed that